packages/common/src/ConfigResponse.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  Address,
+  IntString,
+  NpmLogLevel
+} from './types/Aliases'
+import { Environment } from './Environments'
+
+export interface LoggerConfiguration {
+  logLevel: NpmLogLevel
+  loggerUrl?: string
+  userId?: string
+  applicationId?: string
+}
+
+export interface ConfigResponse {
+  networks: Record<number|string, ConfigEntry>
```

Comment:
And why `number|string`? Do you mean the response would look like
```
"1" : { "bla": "bla bla" },
"mainnet": { "bla": "bla bla" },
"42": { "bla": "bla" },
"rinkeby": { "bla": "bla" }
``` 
???

Reply:
No